cargo/cargo_build_script.bzl

```diff
@@ -450,6 +450,17 @@ def _cargo_dep_env_implementation(ctx):
         outputs = [empty_dir],
         executable = "true",
     )
+
+    build_infos = []
+    if ctx.file.out_dir:
```

Comment:
Should there be some enforcement here that this file is actually the result of [ctx.actions.declare_directory](https://bazel.build/rules/lib/actions#declare_directory)?

Reply:
Done